todo_app/__init__.py

```diff
@@ -0,0 +1,18 @@
+# Set env variables here 
+import requests
+import os
+from dotenv import load_dotenv
+
+host = 'https://api.trello.com'
+api_key = os.getenv('API_KEY')
+api_secret = os.getenv('API_TOKEN')
+
+boards = requests.get( host + '/1/members/me/boards?key=' + api_key + '&token=' + api_secret).json()
+board_id = boards[0]['id']
+
+board_lists = requests.get( host + '/1/boards/' + board_id + '/lists?key=' + api_key + '&token=' + api_secret).json()
+todo_board_list_id = board_lists[0]['id']
```

Comment:
This setup is clever but more complex and less easy to configure than just asking the user to provide their board id/list ids as environment variables; a user with multiple boards has no control over which is used, and possibly not even guarantees that the same board will be used every time the app is started.